app/server/api-generator/src/test/java/io/syndesis/server/api/generator/openapi/OpenApiGeneratorTest.java

```diff
@@ -44,7 +44,7 @@ public void infoShouldHandleNullModels() {
 
         assertThat(summary).isNotNull();
         assertThat(summary.getErrors()).hasSize(1)
-            .allSatisfy(v -> assertThat(v.message()).startsWith("This document cannot be uploaded. Provide an OpenAPI 2.0 document"));
+            .allSatisfy(v -> assertThat(v.message()).startsWith("This document cannot be uploaded. Please provide an OpenAPI document"));
```

Comment:
@TovaCohen I think we said not to say "Please", did I miss remember that?

Reply:
Oh yes I will revert the `Please`